docs/reference/index.asciidoc

```diff
@@ -38,6 +38,8 @@ include::index-modules.asciidoc[]
 
 include::ingest.asciidoc[]
 
+include::autoscaling/index.asciidoc[]
```

Comment:
I think this should go after ILM? I wonder if we initially should move this further down the list?

Reply:
I pushed f940e2de847350421932ab3df036f89e189d3777.